packages/core/src/components/axes/zero-line.ts

```diff
@@ -7,35 +7,43 @@ export class ZeroLine extends Component {
 	type = "zero-line";
 
 	render(animate: boolean) {
+		let axisPosition = this.services.cartesianScales.getRangeAxisPosition();
+		if (this.services.cartesianScales.isDualAxes()) {
+			// get the axis that the groups use
+			axisPosition = this.services.cartesianScales.getDualAxesRangePosition(null, this.configs.groups);
```

Comment:
we don't want to see that `null` in there. Each function ideally would just take a `configs` parameter which'd be a configs object, that'd be way more readable